### PR TITLE
Add tutorial template

### PR DESCRIPTION
**Pull request by agnieszka-m**

### CODE REVIEW

File: tutorials/template.ipynb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    "- **Goal**: After completing this tutorial, you will have achieved/learned about <!--- here state the goal of the tutorial. What will the users have done when they complete it? --->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "<!--- Here provide a short description of the tutorial. What does it teach? What's its expected outcome? --->"
```
Comment (brandenchan) [inline tutorials/template.ipynb:22]: Using this syntax in a markdown box means it doesn't get displayed. I would opt rather to not start and end with the <!--- and just have plain text in each markdown cell

File: tutorials/template.ipynb [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+   "outputs": [],
+   "source": [
+    "# Add some code to run, if applicable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Task 2"
```
Comment (brandenchan) [inline tutorials/template.ipynb:47]: Why don't we show them a numbered list to encourage more step by step instructions?
1.
2.
3.

File: tutorials/template.ipynb [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+   "metadata": {},
+   "source": [
+    "## Task 3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Next Steps\n",
```
Comment (brandenchan) [inline tutorials/template.ipynb:61]: I think it would make more sense to have this at header level 2

**Review by brandenchan — CHANGES_REQUESTED** [review]

**Review by brandenchan — APPROVED** [review]
LGTM

**Review by TuanaCelik — APPROVED** [review]
Looks good to me.
Just a note: Do you already want to add the 'Prerequisites' section to the template now? Or would you rather wait for us to define how we do it ourselves and adding it in here once that's established? 
Ok either way, just wanted to point out. Will approve for now.

**Comment by agnieszka-m** [comment]
@TuanaCelik I'll remove the prereqs section, I added it before finding out what your plan for it was and forgot to remove it after that.

**Pull request merged**